app/controllers/squares_controller.rb

```diff
@@ -23,10 +23,10 @@ def show
     end
     
 
-    # def update 
-    #   @square = Square.find(params[:id])
+    # def update
+    #   @square = Square.find(:id)
```

Comment:
Je veux bien que tu enlève la modif qu'on avait fait pour remettre les params dans le .find() Comme ci-dessous STP
```suggestion
    #   @square = Square.find(params[:id])
```